test/controllers/user.test.ts

```diff
@@ -32,4 +32,25 @@ describe('User class', () => {
             ).toThrowError();
         });
     });
+    describe('generateKeywords()', () => {
+        let user: User;
+
+        beforeEach(() => {
+            user = new User(new JsKeyManager(), jsStorageFactory);
+        });
+
+        it('generates random passphrase words', async () => {
+            const generatedKeywords = await user.generateRandomPassphrase();
```

Comment:
`generateRandomPassphrase()` and `suggestPassphraseWord()` are not asynchronous

Reply:
in user.ts it is asynchronous